backend/src/main/java/ch/puzzle/okr/service/ObjectiveService.java

```diff
@@ -45,6 +45,13 @@ public Objective saveObjective(Objective objective) {
     }
 
     public Objective updateObjective(Long id, Objective objective) {
+        if(objective.getProgress() != null) {
+            List<KeyResult> keyResultList = (List<KeyResult>) this.keyResultRepository.findAll();
+            if(keyResultList.stream().anyMatch(keyResult -> keyResult.getObjective().getId().equals(objective.getId()))) {
+                throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "Can't set the progress of an " +
+                        "objective if you have already defined keyresults!");
```

Comment:
Zeilenumbruch nach ,
und in einem String nicht mit +

Reply:
Zeilenumbruch nun nach "," getrennt